atc/db/build.go

```diff
@@ -149,6 +149,8 @@ type Build interface {
 
 	IsDrained() bool
 	SetDrained(bool) error
+
+	FindWorker(string) (Worker, bool, error)
```

Comment:
I'm not sure if this method should belong on the `Build` interface. It seems to be pretty unrelated to a build and is only using the build's connection to find a worker that matches the name passed in as an argument. Do you think there a better place that this method could live on?

Reply:
@clarafu Thanks for your comment. I also feel not good to put `FindWorker` in `build`. But the problem is, this function needs to be called in step delegates, and a step delegate has only a `build` object that contains db conn, so I cannot find other place to add this function. Do you have any suggestion?

Reply:
No worries, I think this function is no longer needed, I'll delete it in next commit.